packages/shared/src/components/squads/utils.tsx

```diff
@@ -42,3 +47,22 @@ export const ManageSquadPageContainer = classed(
 );
 
 export const ManageSquadPageMain = classed('div', 'flex flex-1 flex-col');
+
+export const isPrivilegedMember = (squad: Squad): boolean =>
+  squad?.currentMember?.permissions?.includes(SourcePermissions.ModeratePost);
```

Comment:
This is not very well named I guess.
It's privileged member but specifically for moderatePost?

Either we should change the function to include `SourcePermissions` as secondary prop (preferred I think)
Or we change the name to `hasModeratePostPermissions`

Reply:
Found we had the `verifyPermission` function which does what you describe, so I removed `isPrivilegedMember` and use that instead